app/services/experimentation/data_export.rb

```diff
@@ -0,0 +1,195 @@
+module Experimentation
+  class DataExport
+    require "csv"
+
+    FOLLOWUP_CUTOFF = 10.days
```

Comment:
I believe this was the cutoff I discussed with Dr. Gupta. We should probably get clarity. At present, only using it to determine when to stop looking for encounters after the experiment is over.